src/chainlit/playground/providers/langchain.py

```diff
@@ -52,8 +52,14 @@ def message_to_string(self, message: PromptMessage) -> str:
     async def create_completion(self, request):
         await super().create_completion(request)
 
-        # Update the request.prompt.messages with the formatted messages
-        self.create_prompt(request)
+        # Edge case where openai doesn't like empty stop sequences
+        if (
+            self.id in ["azure-openai-chat", "openai", "azure", "openai-chat"]
+            and "stop" in request.prompt.settings
+            and request.prompt.settings["stop"] is not None
+            and len(request.prompt.settings["stop"]) == 0
```

Comment:
of stop is an empty array then it works? Also these providers are supposed to be available by default in chainlit

Reply:
good point, I've removed the edge case